backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/utils/VdsCpuUnitPinningHelper.java

```diff
@@ -100,7 +100,8 @@ private void previewPinOfPendingExclusiveCpus(List<VdsCpuUnit> cpuTopology, Map<
                     // Taking offline CPU, should filter out on CpuPinningPolicyUnit.
                     return new ArrayList<>();
                 }
-                vdsCpuUnit.pinVm(vm.getId(), vm.getCpuPinningPolicy());
+                vdsCpuUnit.pinVm(vm.getId(), vm.getCpuPinningPolicy() == CpuPinningPolicy.RESIZE_AND_PIN_NUMA ?
+                        CpuPinningPolicy.MANUAL : vm.getCpuPinningPolicy());
```

Comment:
so how about treating RESIZE_AND_PIN_NUMA as MANUAL in VdsCpuUnit#pinVm instead?